it/JmsRequestReplyClientISpec.scala

```diff
@@ -68,27 +73,42 @@ class JmsRequestReplyClientISpec extends FixtureAsyncFreeSpec with AsyncIOSpec w
   }
 
   "SQS Client" - {
+
     "send a message and handle the reply" in { requestReplyHandler =>
-      val result = sendRequest(requestReplyHandler, "hello 1234")
+      val request =
+        Json.stringify(Json.toJson(GetEditSet(oci = "1", LocalDateTime.of(2023, Month.FEBRUARY, 24, 8, 10))))
+      val expected = Json.stringify(Json.toJson(editSets.editSet1))
 
-      result.asserting(_ mustBe "Echo Server: hello 1234")
+      val result = sendRequest(requestReplyHandler, request)
+      result.asserting(_ mustBe expected)
     }
+
     "send two messages and handle the replies" in { requestReplyHandler =>
```

Comment:
Originally this was meant to ensure that the right response was paired with the right request; now, given that the requests are identical, we don't know that. I'd suggest altering the requests, even slightly.